translator/translate/otel/exporter/awsemf/translator_test.go

```diff
@@ -727,7 +792,61 @@ func TestTranslateAppSignals(t *testing.T) {
 						"app_signals": map[string]any{},
 					},
 				}},
-			want:           testutil.GetConf(t, filepath.Join("appsignals_config_k8s.yaml")),
+			want: map[string]any{
```

Comment:
nit: Instead of redefining all the fields in the file, we should be able to use `conf.Merge` to add the dynamic fields.

Reply:
Ah yeah :(
Lemme see if I can use that.